lib/main.js

```diff
@@ -1,85 +1,64 @@
 'use strict';
 
-var _		= require('lodash');
-var request	= require('request');
-
-var dashboard = require('./oo_dashboard');
-var executions = require('./oo_executions');
-var schedules = require('./oo_schedules');
-var config = require('./oo_config');
-
-/**
- * OO Object declaration
- *
- */
-var OO = {};
-
-/** 
- * OO Object
- *
- * @param {object} options
- * 		required:
- *	 		username - username for OO that is allowed to use the API
- *			password - the password for the username
- *			validSSL - true or false whether to require a valid SSL certificate or allow self-signed certs
- *			baseUrl  - the base url of the REST endpoint for an OO server, for example: http://localhost:8050/oo/rest/v1
- */
-OO.setClient = function(options) {
-
-	if (!options.username || !options.password) {
-		return new Error("no username or password settings provided in options object");
+import 'babel-polyfill';
```

Comment:
Regards the babel transpiler, Babel doesn't support transpiling async/await to ES5 except you include the `babel-polyfill', on the other hand TypeScript inserted the transpile of async/await to ES5 just in new version in the past November.

Regards using the newer version of Node.js (v.7.6), we can do this but most users doesn't tie with latest version of node, and the majority is using LTS version for stability, and linking them to newer version would put them into two hard choices: 1- Upgrade the version which is sensitive for many organization, or to drop this package and replace it with something else.

Regards the import and require syntax, I don't see any differences between them, except for being modern when using import :-)